src/main/java/network/brightspots/rcv/TieBreak.java

```diff
@@ -295,8 +295,8 @@ private String doPreviousRounds(List<String> tiedCandidates) throws TabulationCa
               candidatesInContention.size() > 2 ? "among" : "between",
               Utils.listToSentenceWithQuotes(candidatesInContention),
               tieBreakMode == TieBreakMode.PREVIOUS_ROUND_COUNTS_THEN_RANDOM
-                  ? "random"
-                  : "interactive");
+                  ? TieBreakMode.RANDOM.getInternalLabel()
+                  : TieBreakMode.INTERACTIVE.getInternalLabel());
```

Comment:
Good catch! I wonder if there are any more of these hard-coded strings lurking around in the codebase?

Reply:
I believe not, but I'll check a bit more...